Formula/chadwick.rb

```diff
@@ -14,7 +14,7 @@ class Chadwick < Formula
 
   resource "event_files" do
     url "http://www.retrosheet.org/events/2014eve.zip"
-    sha256 "f2bdcf2c587ea8b67d5c5485edad67c688a8e02325c2b787362ab360b79abdbd"
+    sha256 "0afe0ace917ca9584a5dc1cd7510c82531b3d0544c9ca080099653407edc65f0"
```

Comment:
Looking at the event files it may be possible for us to write short versions of our own and use them for testing, if you wanted to take a swing at that @matteocng.